src/resources/parser/glb-parser.js

```diff
@@ -1145,6 +1144,59 @@ const extensionTransmission = function (data, material, textures) {
     }
 };
 
+const extensionSheen = function (data, material, textures) {
+    material.useSheen = true;
+    if (data.hasOwnProperty('sheenColorFactor')) {
+        const color = data.sheenColorFactor;
+        material.sheen.set(Math.pow(color[0], 1 / 2.2), Math.pow(color[1], 1 / 2.2), Math.pow(color[2], 1 / 2.2));
+    } else {
+        material.sheen.set(1, 1, 1);
+    }
+    if (data.hasOwnProperty('sheenColorTexture')) {
+        material.sheenMap = textures[data.sheenColorTexture.index];
+    }
+    if (data.hasOwnProperty('sheenRoughnessFactor')) {
+        material.sheenGlossiness = data.sheenRoughnessFactor;
+    } else {
+        material.sheenGlossiness = 0.0;
+    }
+    if (data.hasOwnProperty('sheenRoughnessTexture')) {
+        material.sheenGlossinessMap = textures[data.sheenRoughnessTexture.index];
+        material.sheenGlossinessMapChannel = 'a';
+    }
+
+    const sheenGlossChunk = `
+    #ifdef MAPFLOAT
+    uniform float material_sheenGlossiness;
+    #endif
+
+    #ifdef MAPTEXTURE
+    uniform sampler2D texture_sheenGlossinessMap;
+    #endif
+
+    void getSheenGlossiness() {
```

Comment:
any reason not to make this the default sheen glossiness chunk?

Reply:
glTF provides roughness, not glossiness, but I kept it as glossiness to be consistent with the rest of the chunks.

Reply:
Ok, but we are actually looking to -remove- all the custom gltf chunks from gltf-loader (by adding support to standard-material for the custom behaviours).

This way, in future, users will be able to create and modify gltf materials in the editor (which they wouldn't be able to do given these missing features).

So overall I'd suggest either supporting the gltf chunk in standard-material as-is or keep the consistency but expose an option to get this behaviour.

Reply:
Yeah, but the way glTF deals with microsurface descriptions is by using roughness, but all our UI and whatnot is designed for glossiness. 

So unless we want to change so sheen has a Sheen Roughness parameter, instead of Sheen Glossiness like the rest of the parameters, we should do this to not be confusing.

Reply:
Could just be an 'invert' option on the gloss, ccgloss and this one?

Reply:
Yeah, we could have that, but ideally we would never want to invert the glossiness at all. But sure, we could definitely create a PR for having an invert flag on the glossiness in the editor.

Reply:
Do you have any idea why gloss has the `+= 0.000001` term?

Seems this was originally added years ago in https://github.com/playcanvas/engine/pull/1080/files#diff-50e810cc6e8bd1669e9f4411172f8e6f573ba9c1e61bff5401606a09156aa187R24.

Reply:
I think you might divide by gloss, and this is to avoid a div by 0. If you look at sheenD, the first line does 1 / (roughness * roughness) so this is just a way to prevent that from resulting in undefined behaviour.